src/cli/test/bundle_git/test.rb

```diff
@@ -9,20 +9,30 @@ def local_gems
   [name, specs.map{ |spec| spec.version.to_s }.join(',')].join(' ')
 }
 
-# test a github checkout gem
-require 'openstudio'
-require 'openstudio/extension'
-puts OpenStudio::Extension::VERSION
-raise "OpenStudio Extension version does not match" unless OpenStudio::Extension::VERSION.to_s == '0.4.0'
-
+# A gem we do not embed
 require 'tilt'
 puts Tilt::VERSION
 raise "Tilt version does not match" unless Tilt::VERSION == '2.0.8'
 
-did_fail = false
-begin
-  require 'openstudio-standards'
-rescue LoadError
-  did_fail = true
-end
-raise "Should allow to load the embedded openstudio-standards" unless did_fail == false
+# A gem we do embed, different version
+require 'simplecov'
+puts SimpleCov::VERSION
+raise "SimpleCov version does not match" unless SimpleCov::VERSION.to_s == '0.21.2'
+
+# TODO: until we have gems on rubygems, this is unpractical without forcing a
+# download from git for all openstudio-extension, measure-tester, and workflow
+# gems
+
+# # test a github checkout gem
+# require 'openstudio'
+# require 'openstudio/extension'
+# puts OpenStudio::Extension::VERSION
+# raise "OpenStudio Extension version does not match" unless OpenStudio::Extension::VERSION.to_s == '0.8.0'
+
+# did_fail = false
+# begin
+#   require 'openstudio-standards'
+# rescue LoadError
+#   did_fail = true
+# end
+# raise "Should allow to load the embedded openstudio-standards" unless did_fail == false
```

Comment:
Note the comment here.

Reply:
@DavidGoldwasser was working on gettig all of the gems published yesterday. The plan is to have this done prior to an RC2. (maybe today?)